### DISCUSSION TITLE
長照應用科技－新聞心得

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#16] [General]
**內容大綱：**
**智齡科技**執行科技部「人口高齡化議題挑戰計畫」，以「**家人導向雲端照護系統**」為主題，研發適用於華人文化之家人導向照護系統，利用AI來輔助檢視每位長輩的用藥情形，找到可能不正確用藥風險，另外**AI護理紀錄、傷口辨識、智慧排班**等，都是智齡科技因應長照現場開發的AI功能。
智齡科技的第一個產品，專門為**住宿式機構設計的Jubo智慧照護**解決方案，目前全台累積超過150家機構客戶。2020年推出的第二個產品Jubo日照管理解決方案，甚至澎湖離島也有每天使用Jubo的機構客戶。
近期智齡科技發現長照問題不只是台灣遇到的挑戰，也是其他國家的照護痛點。因此已預計**前進海外市場**，設立**北美**分公司，並與全美最大的長照平台Point Click Care合作；與**新加坡**上市公司ECON等養老保健集團接觸並探索合作機會；也順利將Jubo專利照護推車進入到**日本東京**做展示。
**心得感想：**
高齡化已是多數國家面臨的問題，而多數上班族由於平時上班忙碌因此無法能每天照護到家中長者，又會因為在新聞中看到有老人看護欺負老人的疑慮，因此對於照護長者有很大的困擾。我認為該公司發展的功能可以有效改善這個問題，有智慧推車能自動化監測並立即同步，再以記錄數據來分析健康狀況，以科技輔助照護，不僅解決上班族的困擾，又能完善照顧到長者，也能讓照護人員快速以數據了解老人的健康狀況，是對於智慧醫療很好的一個貢獻。
**新聞來源：**
https://www.bnext.com.tw/article/64493/jubo-a-round

**Comment by boson13579** [comment]
高齡長照的問題是不可避免的，隨著科技發展，全球高齡化的趨勢持續前進，但，照顧也便成了輕一輩的勞力負擔，運用Ai能有效減少，勞力成本，也能避免因照護著訓練不足或判斷失誤導致遺憾，隨著大樹鋸含人工智能的出現，一些勞力也能獲得解放，像是台灣的醫療體系一直存在醫病比極高的狀態，這會導致醫療品質降低，也市政府及全名需要面對的問題，有許這間公司的應用有能應用在在醫護照顧，使台灣得醫療品質提升。
